tests/ElasticEngineTest.php

```diff
@@ -606,6 +606,156 @@ public function test_if_the_search_method_with_specified_whereRegexp_clause_buil
         $this->addToAssertionCount(1);
     }
 
+    public function test_if_the_search_method_with_specified_whereGeoDistance_clause_builds_correct_payload()
+    {
+        $this->mockClient()
+            ->shouldReceive('search')
+            ->with([
+                'index' => 'test_index',
+                'type' => 'test_table',
+                'body' => [
+                    'query' => [
+                        'bool' => [
+                            'must' => [
+                                'match' => [
+                                    '_all' => 'flat'
+                                ]
+                            ],
+                            'filter' => [
+                                'bool' => [
+                                    'must' => [
+                                        [
+                                            'geo_distance' => [
+                                                'distance' => 1000,
```

Comment:
Still not sure that you can use it without a unit.

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#distance-units

Wherever distances need to be specified, such as the distance parameter in the Geo Distance Query), the default unit if none is specified is the meter

Reply:
Okay, I'm convinced :)